source/navitiacommon/navitiacommon/models/__init__.py

```diff
@@ -194,6 +194,11 @@ def get_from_key(cls, token, valid_until):
         )
         return query.first()
 
+    @classmethod
+    def get_without_access(cls):
+        query = cls.query.filter(cls.type == 'no_access')
+        return query.first()
```

Comment:
I'm not sure: does this means that we have only one and only one line `no_access` in the DB? If yes, can we enforce this? If we cannot enforce, we might want to document it somewhere at least?

Reply:
Maybe just not allow this in tyr, then (cf. https://github.com/hove-io/navitia/pull/3893#discussion_r1062597105) ?

Reply:
add a filter on login = 'user_without_access'